packages/sku/src/services/webpack/config/plugins/skuWebpackPlugin.ts

```diff
@@ -28,11 +28,10 @@ export class SkuWebpackPlugin implements WebpackPluginInstance {
     validateOptions(options);
 
     this.options = {
-      // Is this default value correct? I imagine it will be set via the options.
```

Comment:
I assume so. If you don't pass anything to `include`, then the plugin won't do anything. Makes sense to me.